config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "modules": ["notebooks"]
```

Comment:
may be a silly question since I am not very advance in python, is there an option to register a module similar way how we currently register stacks or worker handlers? by using an annotation?